.github/workflows/release-admin-tools.yml

```diff
@@ -1,70 +1,40 @@
-name: Release Admin Tools Image
+name: Release AdminTools Image
 
 permissions: read-all
 
 on:
   workflow_dispatch:
     inputs:
       commit:
-        description: "Repo Commit sha"
-        type: string
+        description: "Commit sha"
         required: true
-      patch:
-        description: "The patch version of the admintools image. This is added to the server version"
-        type: string
+      tag:
+        description: "The tag for the new image"
```

Comment:
Can you put a sample in description like in https://github.com/temporalio/docker-compose/blob/main/.github/workflows/create-release.yml#L10.